scripts/kind-e2e/lib_operator_verify_subm.sh

```diff
@@ -379,18 +377,18 @@ function verify_subm_engine_container() {
   grep "SUBMARINER_NATENABLED=$natEnabled" $env_file
   grep "HOME=/root" $env_file
 
-  if kubectl exec $subm_engine_pod_name --namespace=$subm_ns -- command -v command; then
-    # Verify the engine binary is in the expected place and in PATH
-    kubectl exec $subm_engine_pod_name --namespace=$subm_ns -- command -v submariner-engine | grep /usr/local/bin/submariner-engine
+  if kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- command -v command; then
+    # Verify the gateway binary is in the expected place and in PATH
+    kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- command -v submariner-gateway | grep /usr/local/bin/submariner-gateway
 
-    # Verify the engine entry script is in the expected place and in PATH
-    kubectl exec $subm_engine_pod_name --namespace=$subm_ns -- command -v submariner.sh | grep /usr/local/bin/submariner.sh
-  elif kubectl exec $subm_engine_pod_name --namespace=$subm_ns -- which which; then
-    # Verify the engine binary is in the expected place and in PATH
-    kubectl exec $subm_engine_pod_name --namespace=$subm_ns -- which submariner-engine | grep /usr/local/bin/submariner-engine
+    # Verify the gateway entry script is in the expected place and in PATH
+    kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- command -v submariner.sh | grep /usr/local/bin/submariner.sh
+  elif kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- which which; then
+    # Verify the gateway binary is in the expected place and in PATH
+    kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- which submariner-gateway | grep /usr/local/bin/submariner-gateway
```

Comment:
This line shouldn't change

As the binary inside the image remains as submariner-engine yet.

```suggestion
    kubectl exec $subm_gateway_pod_name --namespace=$subm_ns -- which submariner-engine | grep /usr/local/bin/submariner-engine
```

Reply:
Right, we can rename that later after we merge submariner-io/submariner#1160 & submariner-io/shipyard#462